ruleset.md

```diff
@@ -0,0 +1,23 @@
+# Adidas API Guidelines Ruleset
+
+- all JSON fields MUST follow `camelCase`
+- field names MUST be ASCII alphanumeric characters or `_` or `$`
+- collection/array fields MUST have names in plural
+- all requests go through `https` protocol
+- every API operation MUST have at least one `2xx` response
+- `GET` request MUST NOT accept a `body` parameter
+- all success responses MUST be of media type `application/hal+json`

```

Comment:
+ what we have discussed with async task (problem detail might be legit)

Reply:
yeah. That however, will affect the guidelines as well. How would you word this then?

Reply:
> ... with the exception of asynchronous task tracking resource response where problem detail is permitted to communicate the details about the failed task in the tracking baton response.

Reply:
But I’ve got feeling that we are shooting ourselves in the back with this exception...

Reply:
I think the most reasonable thing to do is to set a warning instead of error level of severity. I don't see how would you statically check (in OAS+Spectral at least) whether a request is async